docs/references/faq.md

```diff
@@ -13,23 +13,37 @@ There are many benefits to this—not least of which is cost management—so [fo
 
 ## How does this work with—or replace—my existing Prometheus?
 
+Opstrace is essentially a horizontally scalable, highly available Prometheus with inexpensive long-term storage.
```

Comment:
> with inexpensive long-term storage

I read this as Opstrace will charge me for storage.

Reply:
Thanks, I've clarified this.